errutil/error.go

```diff
@@ -0,0 +1,19 @@
+// Package errutil implements some useful extensions to the stdlib
+// Go errors package, in the same spirit of packages like ioutil/httputil.
+//
+// Utilities include:
+//
+// - An error type that makes it easy to work with const error sentinels.
+// - Improved chaining of error sentinels (wrapping multiple errors).
```

Comment:
this is not implemented yet.

Reply:
Yeah I though it would be OK to add it already, but I can remove =P

Reply:
np, it's your call, just because in the meantime the docs will be wrong.

Reply:
I think I was too relaxed...will remove and then we can add it back later (wrong docs are bad docs =P)